components/epaxos/src/snapshot/rocks/engine.rs

```diff
@@ -26,30 +26,56 @@ impl RocksDBEngine {
         Ok(RocksDBEngine { db: db })
     }
 
+    /// make rocksdb column family handle
+    fn _make_cf_handle(&self, cf: &DBColumnFamily) -> Result<&CFHandle, Error> {
+        match self.db.cf_handle(cf.as_str()) {
+            Some(h) => Ok(h),
+            None => Err(Error::RocksDBError {
+                msg: format!("got column family {} handle failed", cf.as_str()),
+            }),
+        }
+    }
+
     /// Set a key-value pair to rocksdb.
     fn _set(&mut self, cf: &DBColumnFamily, k: &[u8], v: &[u8]) -> Result<(), Error> {
-        let cfh = self.db.cf_handle(cf.as_str()).unwrap();
+        let cfh = self._make_cf_handle(cf)?;
         Ok(self.db.put_cf(cfh, k, v)?)
     }
 
     /// Get a value from rocksdb with it's key.
     fn _get(&self, cf: &DBColumnFamily, k: &[u8]) -> Result<Vec<u8>, Error> {
-        let cfh = self.db.cf_handle(cf.as_str()).unwrap();
+        let cfh = self._make_cf_handle(cf)?;
 
         match self.db.get_cf(cfh, k) {
             Ok(option_val) => match option_val {
                 Some(val) => {
                     return Ok(val.to_vec());
                 }
                 None => {
+                    let k_str = match str::from_utf8(k) {
+                        Ok(s) => s,
+                        Err(err) => {
+                            return Err(Error::RocksDBError {
+                                msg: format!("got error {} while convert utf8 to str", err),
```

Comment:
got error 去掉吧...感觉在消息里没啥用...
```suggestion
                                msg: format!("{} while converting utf8 to str", err),
```